pkgs/development/python-modules/ropgadget/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, capstone
+}:
+
+buildPythonPackage rec {
+  pname = "ropgadget";
+  version = "6.6";
+
+  src = fetchFromGitHub {
+    owner = "JonathanSalwan";
+    repo = "ROPgadget";
+    rev = "v${version}";
+    sha256 = "1i0gx0cwhxk6d8byvck17hh83szz3k6ndd118ha3q0r0msap0lz1";
+  };
+
+  propagatedBuildInputs = [
+    capstone
+  ];
+
+  # Test suite is working with binaries
```

Comment:
You mean it needs to download them or what exactly prevents us from running the tests?

Reply:
We would need the [test data](https://github.com/JonathanSalwan/ROPgadget/tree/master/test-suite-binaries). It looks like that it would be possible to get the tes running with some "bash magic".